Makefile

```diff
@@ -391,6 +391,7 @@ shot:
 		radare.org:/srv/http/radareorg/get/shot
 
 tests test:
+	export LC_ALL=C
```

Comment:
That line those nothing. The very next line wont maintain the same environment

also its wrong to override user settings. If you have warnjngs fix your system

Reply:
It is to ensure that local tests of everyone running tests have same setting.

Reply:
no. this line is useless. try this if my epxlanation above wasnt clear

```
$ make
export JEJE=123
echo $JEJE

$ cat Makefile
all:
	export JEJE=123
	echo $$JEJE
$
```

Reply:
ah ok